etc/spec/SPECIFICATION.md

```diff
@@ -89,22 +89,23 @@ username.
 If a user wants to keep their username, they have to enter a password by
 editing their user account through the Settings screen.
 
-The Login screen can be either a Simple Login screen or a Password Login
-screen. The Simple Login screen is shown by default.
+## Login
 
-## Simple Login
+The Login screen consists of only a modal window with the heading
+"Ting", a textbox, a button and a password form if that username is registered.
 
-The Simple Login screen consists of only a modal window with the heading
-"Ting", a textbox, a button, and a link.
+![Login screen](http://i.imgur.com/wYUj1ev.jpg)
+*Image: Simple Login*
 
-![Login screen](http://i.imgur.com/FirJeIL.jpg)
-*Image: The simple login screen*
+![Login screen](http://i.imgur.com/xLVm6eV.jpg?1)
+*Image: Password Login*
 
-The placeholder text in the textbox is "Διάλεξε ένα ψευδώνυμο". The textbox is
+The placeholder text in the textbox is "Γράψε ένα ψευδώνυμο". The textbox is
 focused by default when the screen is visited. The placeholder text is hidden
-when the user starts typing some text, unless the text is cleared. Pressing the
-Enter key when the keyboard is focused on the textbox is equivalent to pressing
-the button.
+when the user starts typing some text, unless the text is cleared. In the same
+way works the placeholder for the password box "Γράψε τον κωδικό" that appears
+only if that username is registered. Pressing the Enter key when the keyboard is
```

Comment:
Not sure how this will work. Let's say I want to type the username "dionyziz" and the username "dion" is registered. Will the password field appear while I am typing and then disappear again? That doesn't sound like a good UI. Or are you thinking of something else?